models/sipnet/Dockerfile

```diff
@@ -48,13 +47,10 @@ FROM pecan/models:${IMAGE_VERSION}
 # SETUP FOR SPECIFIC SIPNET VERSION
 # ----------------------------------------------------------------------
 
-# Some variables that can be used to set control the docker build
-ARG MODEL_VERSION=136
-
 # Setup model_info file
 COPY model_info.json /work/model.json
 RUN sed -i -e "s/@VERSION@/${MODEL_VERSION}/g" \
            -e "s#@BINARY@#/usr/local/bin/sipnet.${MODEL_VERSION}#g" /work/model.json
 
 # COPY sipnet binary
-COPY --from=sipnet-binary /src/sipnet /usr/local/bin/sipnet.${MODEL_VERSION}
+COPY --from=sipnet-binary /src/sipnet/sipnet /usr/local/bin/sipnet.${MODEL_VERSION}
```

Comment:
Is this a typo, or does the model install itself into a `sipnet` subdirectory of its own repo?

Reply:
nope, the model when compiled creates a binary that is called sipnet in the sipnet folder. When finished compiling I will copy the binary from one container to another container using this copy command.

Reply:
Ah, OK, thanks!